resource/storage.go

```diff
@@ -20,7 +20,7 @@ type Storer interface {
 	// If the fetching of the data is not immediate, the method must listen for cancellation
 	// on the passed ctx. If the operation is stopped due to context cancellation, the
 	// function must return the result of the ctx.Err() method.
-	Find(ctx context.Context, lookup *Lookup, page, perPage int) (*ItemList, error)
+	Find(ctx context.Context, lookup *Lookup, page, perPage int, skip int) (*ItemList, error)
```

Comment:
first I started with `skip` then decided to use `offset` since this is a better term when doing queries and people know what it is meant to be. but then I just changed the query param to offset and left `skip` in all functions as is.

I am just about to push the PR. do you want `skip` or `offset` as param in the funcs?